bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/Converter.java

```diff
@@ -0,0 +1,189 @@
+        } else {
+            System.err.println("python");
+            return null;
+        }
+
+    }
+
+    public CompleteDFD assToDFD(String inputModel, String inputFile, String modelLocation, String outputFile) {
+        final var usageModelPath = Paths.get("models", inputModel, inputFile + ".usagemodel").toString();
+        final var allocationPath = Paths.get("models", inputModel, inputFile + ".allocation").toString();
+        final var nodeCharPath = Paths.get("models", inputModel, inputFile + ".nodecharacteristics").toString();
+
+        DataFlowConfidentialityAnalysis analysis = new PCMDataFlowConfidentialityAnalysisBuilder().standalone().modelProjectName(modelLocation)
+                .usePluginActivator(Activator.class).useUsageModel(usageModelPath).useAllocationModel(allocationPath)
+                .useNodeCharacteristicsModel(nodeCharPath).build();
+
+        analysis.initializeAnalysis();
+        analysis.findAllSequences();
+        var sequences = analysis.findAllSequences();
+        var propagationResult = analysis.evaluateDataFlows(sequences);
+
+        ProcessASS ass2dfd = new ProcessASS();
+
+        return ass2dfd.transform(propagationResult);
+    }
+
+    public int runPythonScript(String in, String format, String out) {
+        String[] command = {"python3", "convert_model.py", in, format, "-op", out};
+
+        ProcessBuilder processBuilder = new ProcessBuilder(command);
+        Process process;
+        try {
+            process = processBuilder.start();
+            return process.waitFor();
+        } catch (IOException | InterruptedException e) {
+            e.printStackTrace();
+        }
+        return -1;
+    }
+
+    public void store(DFD web, String outputFile) {
+        objectMapper = new ObjectMapper();
+        objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        objectMapper.enable(SerializationFeature.INDENT_OUTPUT);
+
+        try {
+            objectMapper.writeValue(new File(outputFile), web);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void store(CompleteDFD complete, String outputFile) {
+        ResourceSet rs = new ResourceSetImpl();
+        Resource dfdResource = createAndAddResource(outputFile + ".dataflowdiagram", new String[] {"dataflowdiagram"}, rs);
+        Resource ddResource = createAndAddResource(outputFile + ".datadictionary", new String[] {"datadictionary"}, rs);
+
+        dfdResource.getContents().add(complete.dataFlowDiagram());
+        ddResource.getContents().add(complete.dataDictionary());
+
+        saveResource(dfdResource);
+        saveResource(ddResource);
+
+    }
+    
+    public MicroSecEnd loadMicro(String inputFile) {
+        objectMapper=new ObjectMapper();
+        file = new File(inputFile + ".json");
+        try {
+            return objectMapper.readValue(file, MicroSecEnd.class);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
This prints to System.err, please replace this with an appropriate logger message. Also returning null here is fishy